### ISSUE TITLE
Bug when using field in draft

### ISSUE THREAD

**Issue by etx** [pixelandtonic/StoreHours#6]
When loading up a draft of an entry it looks like ``prepValueFromPost`` and ``prepValue`` methods are both run, when causes the open/close properties to become empty strings. 

Below is my quick hack, but I suck at php so it's probably not a good fix. 

```
foreach ($value as &$day)
			{
				if ((is_string($day['open']) && $day['open']) || (is_array($day['open']) && $day['open']['time']))
				{
					$day['open'] = DateTime::createFromString($day['open'], $timezone);
				}
				else if( !($day['open'] instanceof DateTime) )
				{
					$day['open'] = '';
				}

				if ((is_string($day['close']) && $day['close']) || (is_array($day['close']) && $day['close']['time']))
				{
					$day['close'] = DateTime::createFromString($day['close'], $timezone);
				}
				else if( !($day['close'] instanceof DateTime) )
				{
					$day['close'] = '';
				}
			}
```

**Comment by takobell** [comment]
Resolved in d7e40361d9c37

**Comment by jsunsawyer** [comment]
Ahhhh. I found this when trying to figure out why my client's Store Hours fields were being reset.

I originally checked to see if the plugin was up to date, and thought it was, but the plugin version number didn't change when this issue was resolved. Hope this helps someone else!
